tests/net/mld/src/main.c

```diff
@@ -330,42 +331,58 @@ static void send_query(struct net_if *iface)
 	zassert_not_null(pkt, "Cannot allocate pkt");
 
 	net_pkt_set_ipv6_hop_limit(pkt, 1); /* RFC 3810 ch 7.4 */
-	net_ipv6_create(pkt, &peer_addr, &dst);
+	ret = net_ipv6_create(pkt, &peer_addr, &dst);
+	zassert_false(ret, "Cannot create ipv6 pkt");
```

Comment:
Can you please use proper boolean expressions here and below, like `ret != 0`, `ret == 0`, `ret < 0` (whatever actually applies to a particular case).

Reply:
Probably not worth it, we are only interested about if the code succeed (returns value 0) or not. I am fine with current solution.

Reply:
Yeah, that's just test code...